set.mm

```diff
@@ -72652,7 +72652,7 @@ Power Set ( ~ ax-pow ).  (Contributed by Mario Carneiro, 20-May-2013.)
     dmmpt2g.f $e |- F = ( x e. A , y e. B |-> C ) $.
```

Comment:
dmmpog.f

Reply:
I guess you are reviewing commit by commit? Because I think all four of the hypotheses you mention were renamed, just in later commits in this pull request.

In the "Files changed" tab I see names containing `mpo` not `mpt2`

Reply:
`I guess you are reviewing commit by commit`  Yes